### PR TITLE
fix antithesis trigger

### PR DESCRIPTION
**Pull request by barisoyoruk**
## General
**Before this PR**:

**After this PR**:

==COMMIT_MSG==
==COMMIT_MSG==

**Priority**:

**Concerns / possible downsides (what feedback would you like?)**:

**Is documentation needed?**:

## Compatibility
**Does this PR create any API breaks (e.g. at the Java or HTTP layers) - if so, do we have compatibility?**:

**Does this PR change the persisted format of any data - if so, do we have forward and backward compatibility?**:

**The code in this PR may be part of a blue-green deploy. Can upgrades from previous versions safely coexist? (Consider restarts of blue or green nodes.)**:

**Does this PR rely on statements being true about other products at a deployment - if so, do we have correct product dependencies on these products (or other ways of verifying that these statements are true)?**:

**Does this PR need a schema migration?**

## Testing and Correctness
**What, if any, assumptions are made about the current state of the world? If they change over time, how will we find out?**:

**What was existing testing like? What have you done to improve it?**:

**If this PR contains complex concurrent or asynchronous code, is it correct? The onus is on the PR writer to demonstrate this.**:

**If this PR involves acquiring locks or other shared resources, how do we ensure that these are always released?**:

## Execution
**How would I tell this PR works in production? (Metrics, logs, etc.)**:

**Has the safety of all log arguments been decided correctly?**:

**Will this change significantly affect our spending on metrics or logs?**:

**How would I tell that this PR does not work in production? (monitors, etc.)**:

**If this PR does not work as expected, how do I fix that state? Would rollback be straightforward?**:

**If the above plan is more complex than “recall and rollback”, please tag the support PoC here (if it is the end of the week, tag both the current and next PoC)**:

## Scale
**Would this PR be expected to pose a risk at scale? Think of the shopping product at our largest stack.**:

**Would this PR be expected to perform a large number of database calls, and/or expensive database calls (e.g., row range scans, concurrent CAS)?**:

**Would this PR ever, with time and scale, become the wrong thing to do - and if so, how would we know that we need to do something differently?**:

## Development Process
**Where should we start reviewing?**:

**If this PR is in excess of 500 lines excluding versions lock-files, why does it not make sense to split it?**:

**Please tag any other people who should be aware of this PR**:
@jeremyk-91
@sverma30
@raiju

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -139,7 +139,7 @@ jobs:
       - run:
           command: |
             # hack - notion of "owners" isn't supported in Circle 2
-            if [ $CIRCLE_PROJECT_USERNAME = 'palantir' ] && [ -z $CIRCLE_PR_NUMBER ] && [ $CIRCLE_TAG ]; then
```
Comment (barisoyoruk) [inline .circleci/config.yml:142]: We want to trigger with each commit, not each release

File: scripts/circle-ci/trigger-antithesis-simulation.sh [hunk 2]
```diff
@@ -5,6 +5,17 @@ if [[ -z "${ANTITHESIS_WEBHOOK_PASSWORD}" ]]; then
   exit 1
 fi
 
-WEBHOOK_LOCATOR="palantir_atlasdb__baseline__latest"
+if [[ -z "${ANTITHESIS_REPORT_RECIPIENT}" ]]; then
```
Comment (barisoyoruk) [inline scripts/circle-ci/trigger-antithesis-simulation.sh:8]: Probably this is sensitive?

Comment (barisoyoruk) [reply scripts/circle-ci/trigger-antithesis-simulation.sh:8]: I added this to the circle ci

File: scripts/circle-ci/trigger-antithesis-simulation.sh [hunk 3]
```diff
@@ -5,6 +5,17 @@ if [[ -z "${ANTITHESIS_WEBHOOK_PASSWORD}" ]]; then
...
+fi
+
+WEBHOOK_LOCATOR="atlasdb"
 echo "Triggering simulation on Antithesis via the ${WEBHOOK_LOCATOR} webhook."
-curl -v -u "palantir:${ANTITHESIS_WEBHOOK_PASSWORD}" -X POST https://palantir.antithesis.com/api/v1/launch_experiment/${WEBHOOK_LOCATOR}
+curl -v -u "palantir:${ANTITHESIS_WEBHOOK_PASSWORD}" -X POST https://palantir.antithesis.com/api/v1/launch_experiment/${WEBHOOK_LOCATOR} -d \
+'{ "params": {
+    "antithesis.images":"palantirtechnologies/cassandra:latest;palantirtechnologies/timelock-server-distribution:latest;palantirtechnologies/atlasdb-workload-server-distribution:latest",
+    "antithesis.config_image":"palantirtechnologies/atlasdb-workload-server-antithesis:latest",
+    "custom.duration":"3",
```
Comment (mdaudali) [inline scripts/circle-ci/trigger-antithesis-simulation.sh]: What led you to choose 3 (hours) down from the current 12?

**Comment by mdaudali** [comment]
Discussed offline - atm, 12 hours per commit is likely too much given the rate of commits. We'll do a different amount of hours for "normal" commits vs release commits and try that out.

File: scripts/circle-ci/trigger-antithesis-simulation.sh [hunk 4]
```diff
@@ -5,6 +5,18 @@ if [[ -z "${ANTITHESIS_WEBHOOK_PASSWORD}" ]]; then
...
-WEBHOOK_LOCATOR="palantir_atlasdb__baseline__latest"
-echo "Triggering simulation on Antithesis via the ${WEBHOOK_LOCATOR} webhook."
-curl -v -u "palantir:${ANTITHESIS_WEBHOOK_PASSWORD}" -X POST https://palantir.antithesis.com/api/v1/launch_experiment/${WEBHOOK_LOCATOR}
+if [[ -z "${ANTITHESIS_REPORT_RECIPIENT}" ]]; then
+  echo "Antithesis report recipient is not set as an environment variable, exiting."
+  exit 1
+fi
+
+WEBHOOK_LOCATOR="atlasdb"
+TEST_DURATION=$([ -n "${CIRCLE_TAG}" ] && echo "12" || echo "3")
```
Comment (barisoyoruk) [inline scripts/circle-ci/trigger-antithesis-simulation.sh:14]: I tested and it looks like it is working

**Review by mdaudali — APPROVED** [review]

**Pull request merged**